spec/models/profile_spec.rb

```diff
@@ -1128,8 +1128,8 @@
       # controller's params. As this is a model spec, we have to fake
       # the params object.
       fake_params = ActionController::Parameters.new(
-        user: OpenStruct.new(id: 'fake_user_id'),
```

Comment:
I feel like we could add an exception to this rule for specs if we wanted?

Reply:
I kinda feel like most of these examples are improved (made more realistic) by not using structs?

Reply:
Yeah, that was my thought as well. We also have `double()` which is not _quite_ the same as `OpenStruct`, but appropriate enough for testing cases where we need something that responds to a given method.

Reply:
I also meant to mention that I'm much less concerned about using it in specs though, yeah.